jenkins/scripts/dynamic_worker_workflow/run_clean.sh

```diff
@@ -12,9 +12,9 @@ fi
 if [[ "${REPO_NAME}" == "metal3-dev-env" ]] ||
    [[ "${REPO_NAME}" == "cluster-api-provider-metal3" ]] \
     ; then
-    pushd tested_repo
+    pushd "/home/${USER}/tested_repo"
```

Comment:
nit: you could use `${HOME}` instead of `/home/${USER}` to make it shorter and also cover cases when the home folder is not in /home.

Reply:
done